### PR TITLE
Chore/upgrade mobx

### PR DESCRIPTION
**Pull request by wodenx**

### CODE REVIEW

File: packages/bodiless-core/src/components/PageEditor.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ const GlobalContextMenu: FC<Props> = observer(() => {
   const options = contextMenuOptions.filter(
     (op: TMenuOption) => op.global !== false,
   );
+  console.log('gcm', options);
```
Comment (lucas-varela) [inline packages/bodiless-core/src/components/PageEditor.tsx:52]: Are these two logs needed?

File: packages/bodiless-core/src/components/PageEditor.tsx [hunk 2]
```diff
@@ -87,13 +88,17 @@ const PageEditor: FC<Props> = ({ children, ui }) => {
...
+  console.log('pageditor');
 
   // Register buttons to the main menu.
   useRegisterMenuOptions({
     getMenuOptions,
     name: 'Editor',
   });
-  useEffect(() => { if (!context.isActive) context.activate(); }, []);
+  useEffect(() => {
+    console.log('effect', context, context.isActive);
```
Comment (lucas-varela) [inline packages/bodiless-core/src/components/PageEditor.tsx:99]: This one also.

File: packages/fclasses/src/flowHoc.tsx [hunk 3]
```diff
@@ -40,9 +40,15 @@ function mergeMeta(objValue:any, srcValue:any) {
...
-  const NewComponent = hoc(Component) as ComponentWithMeta<Q>;
-  const finalMeta = mergeWith({}, Component, NewComponent, mergeMeta);
-  return Object.assign(NewComponent, finalMeta);
+  try {
+    const NewComponent = hoc(Component) as ComponentWithMeta<Q>;
+    const finalMeta = mergeWith({}, Component, NewComponent, mergeMeta);
+    return Object.assign(NewComponent, finalMeta);
+  } catch (e) {
+    console.log(e);
+    console.log(Component);
```
Comment (lucas-varela) [inline packages/fclasses/src/flowHoc.tsx:49]: More logs here.

File: packages/gatsby-theme-bodiless/src/dist/Page.tsx [hunk 4]
```diff
@@ -71,6 +71,7 @@ const ShowDesignKeys = (
 const Page: FC<PageProps> = observer(({ children, ui, ...rest }) => {
   const { PageEditor: Editor, ContextWrapper: Wrapper } = getUI(ui);
   if (process.env.NODE_ENV === 'development') {
+    console.log('edit page');
```
Comment (lucas-varela) [inline packages/gatsby-theme-bodiless/src/dist/Page.tsx:74]: Another log.

File: sites/test-site/src/data/site/MainMenu$4d51d3e5-3c89-41e6-8970-13a4b1c7b7f5$title$text.json [hunk 5]
```diff
@@ -1,3 +1,3 @@
 {
-  "text": "Tutorials"
-}
+  "text": "Tutodfgrials"
```
Comment (lucas-varela) [inline sites/test-site/src/data/site/MainMenu$4d51d3e5-3c89-41e6-8970-13a4b1c7b7f5$title$text.json:2]: Was this meant to be changed?

**Review by lucas-varela — CHANGES_REQUESTED** [review]

File: packages/bodiless-core/src/components/PageEditor.tsx (hunk 1 above)
Comment (Guilherme-Almeida-Zeni) [reply packages/bodiless-core/src/components/PageEditor.tsx:52]: @lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!

File: packages/bodiless-core/src/components/PageEditor.tsx [hunk 6]
```diff
@@ -87,13 +88,17 @@ const PageEditor: FC<Props> = ({ children, ui }) => {
   };
 
   const { PageOverlay = () => null } = newUI;
+  console.log('pageditor');
```
[the remark this answers is not in the document]
Comment (Guilherme-Almeida-Zeni) [reply packages/bodiless-core/src/components/PageEditor.tsx:91]: @lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!

File: packages/bodiless-core/src/components/PageEditor.tsx (hunk 2 above)
Comment (Guilherme-Almeida-Zeni) [reply packages/bodiless-core/src/components/PageEditor.tsx:99]: @lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!

File: packages/fclasses/src/flowHoc.tsx (hunk 3 above)
Comment (Guilherme-Almeida-Zeni) [reply packages/fclasses/src/flowHoc.tsx:49]: @lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!

File: packages/gatsby-theme-bodiless/src/dist/Page.tsx (hunk 4 above)
Comment (Guilherme-Almeida-Zeni) [reply packages/gatsby-theme-bodiless/src/dist/Page.tsx:76]: @lucas-varela , fixed as part of https://github.com/wodenx/Bodiless-JS/pull/23. Thanks!

**Pull request merged**